### DISCUSSION TITLE
Can't find version control settings

### DISCUSSION THREAD

**Discussion by maxx27** [sbarex/SourceCodeSyntaxHighlight#250] [Q&A] [answered]
I'm using syntax-highlight v2.1.19. In the README, it is written that

> You can also enable the support of common VCS to highlight the added/edited/removed lines from last commit.

But I can't find this setting 🫤. My general tab doesn't contain it:

![image](https://github.com/sbarex/SourceCodeSyntaxHighlight/assets/5478638/75ae97c1-4c7d-476f-b8b5-f8f579b60fcf)

Could you please point out where the setting is located?

**Comment by sbarex** [comment, marked as answer]
From the view menu, flag the `show advanced settings` item.
